pkg/crc/constants/constants.go

```diff
@@ -121,6 +120,13 @@ func InstanceName(preset crcpreset.Preset) string {
 	return fmt.Sprintf("crc-%s", preset.String())
 }
 
+func GetKubeconfigFilePath(preset crcpreset.Preset) string {
+	if preset != crcpreset.Podman {
+		return filepath.Join(MachineInstancesDir, InstanceDirName(preset), "kubeconfig")
+	}
+	return ""
```

Comment:
podman was not special cased before, why the change? Should this go to a commit of its own maybe?

Reply:
podman doesn't require this helper, i've removed the check and kept it like it was before